server/conf/application.conf

```diff
@@ -585,6 +585,12 @@ api_applications_list_max_page_size = ${?CIVIFORM_API_APPLICATIONS_LIST_MAX_PAGE
 ## Feature flag toggles.
 # Launched features.
 
+# When set to true Global Admins can also be Program Admins.
+# It will enable Global Admin assign themselves as Program Admins, and get access to 
+# all Applications for that program.
+allow_global_admins_be_program_admins = false
```

Comment:
Not blocking: In a subsequent PR, we'll need to add this environment variable to the terraform config to support piping it all the way through to the server. Just calling out explicitly since there's a follow-on item to be done (which I'm sure you already know) :).

Reply:
yup. there's probably more flags we need to put in terraform. Added https://github.com/seattle-uat/civiform/issues/2932.